jcl/src/jdk.management/share/classes/com/ibm/lang/management/ProcessorUsage.java

```diff
@@ -99,7 +99,7 @@ private ProcessorUsage(long user, long system, long idle, long wait,
 			throw new IllegalArgumentException("id(" + id + ") < -1"); //$NON-NLS-1$ //$NON-NLS-2$
 		}
 		if ((online < 0) || (online > 1)) {
-			throw new IllegalArgumentException("online(" + online + ") < 0) || online > 1"); //$NON-NLS-1$ //$NON-NLS-2$
+			throw new IllegalArgumentException("online(" + online + ") < 0 || online > 1"); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
Perhaps a better message would be 'online(N) not in [0,1]'?

Reply:
Updated.